DOCS/waf-buildsystem.rst

```diff
@@ -17,20 +17,20 @@ shortcomings:
    When adding a new feature using the old configure, one had to add a fair
    amount of code to the shell script to do option parsing, detection of the
    feature and declaration of variables for the Makefile to pickup. The worst
-   part is this pieces are spread apart in the configure and copy pasted for
-   any single case. That brings us to..
+   part is these pieces are spread apart in the configure and copy-pasted for
+   any single case. That brings us to...
 
 2) --enable-feature has to be overridden by the user and helps them understand that
    they have libraries missing and should install them for the feature to be
    enabled.
 
-3) Must be customizable, hackable, pleasant to the developer eyes and to work
-   with in general.
+3) Must be customizable, hackable, pleasant to the developer's eyes and to work
+   within general.
```

Comment:
This is wrong on two accounts:

1. Should be `developers'`, for consistency with your other plural possessive.
2. `within` is actually `with in`, or possibly `with, in` for clarity.

Reply:
Oops, it's really not `within` what a mistake...